tools/codegen/dest/register_dispatch_key.py

```diff
@@ -312,14 +390,14 @@ def gen_class_set_output_body(self, k: SchemaKind) -> str:
     "Expected out tensor to have dtype ", options.dtype(), ", but got ", out.dtype(), " instead");
 TORCH_CHECK(options.device() == out.device(),
     "Expected out tensor to have device ", options.device(), ", but got ", out.device(), " instead");
-bool resized = at::native::resize_output(outputs_[output_idx], sizes);
+bool resized = {self.cpp_namespace}::resize_output(outputs_[output_idx], sizes);
```

Comment:
This seems a little questionable; are you actually changing what namespace we reference `resize_output` here?

Reply:
Yep that's a bug, thanks! Since `at::native::resize_output` is the thing we want to call to resize the output tensor, regardless of which backend we're generating. We wouldn't have hit it until we eventually tried to enable structured external kernels.

Reply:
Also for the empty calls above, I think